contracts/modules/dispute/policies/ArbitrationPolicySP.sol

```diff
@@ -77,6 +101,9 @@ contract ArbitrationPolicySP is IArbitrationPolicy, AccessManagedUpgradeable, UU
         if (decision) {
             (, address disputeInitiator, , , , , ) = IDisputeModule(DISPUTE_MODULE).disputes(disputeId);
             IERC20(PAYMENT_TOKEN).safeTransfer(disputeInitiator, ARBITRATION_PRICE);
+        } else {
+            ArbitrationPolicySPStorage storage $ = _getArbitrationPolicySPStorage();
+            IERC20(PAYMENT_TOKEN).safeTransfer($.treasury, ARBITRATION_PRICE);
```

Comment:
What's the rationale for transferring the entire arbitration cost to the treasury?